apps/api-extractor/src/generators/ApiReportGenerator.ts

```diff
@@ -154,7 +154,8 @@ export class ApiReportGenerator {
     // Write the closing delimiter for the Markdown code fence
     stringWriter.writeLine('\n```');
 
-    return stringWriter.toString();
+    // Remove any trailing spaces
+    return stringWriter.toString().replace(/ +$/gm, '');
```

Comment:
(Maybe define `private static _TrimSpacesRegExp = / +$/gm;`.  We generally do this based on [a superstition](https://stackoverflow.com/questions/14352100/does-v8-cache-compiled-regular-expressions-automatically) that it increases likelihood of caching the compiled RegEx.)